src/test/java/edu/ucla/library/prl/harvester/LdapAuthIT.java

```diff
@@ -34,15 +49,59 @@ public class LdapAuthIT {
      */
     @Test
     public void testLdapConnection(final Vertx aVertx, final VertxTestContext aContext) {
-        final String ldapURL = System.getenv(Config.LDAP_URL); // If null, auth creation will fail
-        final LdapAuthenticationOptions ldapOpts = new LdapAuthenticationOptions().setUrl(ldapURL)
-                .setAuthenticationQuery("cn={0},ou=prlgroup,dc=glauth,dc=com");
-        final AuthenticationProvider auth = LdapAuthentication.create(aVertx, ldapOpts);
+        final AuthenticationProvider auth = LdapAuthnProvider.create(aVertx, AUTH_OPTS);
 
         // Testing password, used during container-tested build, is hard-coded (for now?)
-        auth.authenticate(new UsernamePasswordCredentials("prl", "pearl")).onSuccess(user -> {
-            // Just a "hello world" test for now...
+        auth.authenticate(new UsernamePasswordCredentials(USERNAME, PASSWORD)).onSuccess(user -> {
             aContext.completeNow();
         }).onFailure(aContext::failNow);
     }
+
+    /**
+     * A test that confirms LDAP user authentication works.
+     *
+     * @param aVertx A Vert.x instance
+     * @param aContext A testing context
+     */
+    @Test
+    public void testLdapUser(final Vertx aVertx, final VertxTestContext aContext) {
+        final AuthenticationProvider auth = LdapAuthnProvider.create(aVertx,
+                AUTH_OPTS.setUserQuery(System.getenv(Config.LDAP_AUTH_QUERY), new ArrayList<>()));
```

Comment:
Was this meant to reference `LDAP_USER_QUERY` (which is commented out in the POM)?

Reply:
Yes, thanks. In this case, they both work, apparently, but it should be using `LDAP_USER_QUERY` as you note.